lockfile.json

```diff
@@ -248,6 +248,9 @@
   "slint": {
     "revision": "0d4dda94f96623302dfc234e06be62a5717f47da"
   },
+  "solidity": {
+    "revision": "defd9e5a2aef4e6fd848d0a7e779400747bc45dd"
```

Comment:
This hash is for the master branch, the correct branch should be `with-generated-c-code` (52ed0880c0126df2f2c7693f215fe6f38e4a2e0a): https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lua/nvim-treesitter/parsers.lua#L965

Reply:
Ah, then the update lockfile PR is not smart enough to check the right branch. We need to fix that or disable automatic updates. I'll fix this manually for this PR.